engine.py

```diff
@@ -694,21 +692,24 @@ def show_dialog(self, title, *args, **kwargs):
 
         :returns: the created widget_class instance
         """
-        if not sgtk.util.is_macos():
-            return super(MayaEngine, self).show_dialog(title, *args, **kwargs)
-        else:
-            if not self.has_ui:
-                self.log_error(
-                    "Sorry, this environment does not support UI display! Cannot show "
-                    "the requested window '%s'." % title
-                )
-                return None
+        from sgtk.platform.qt import QtCore, QtGui
 
-            from sgtk.platform.qt import QtCore, QtGui
+        if not self.has_ui:
+            self.log_error(
+                "Sorry, this environment does not support UI display! Cannot show "
+                "the requested window '%s'." % title
+            )
+            return None
```

Comment:
Minor point no need for "None" when returning nothing. This is Python's default.
```diff
-               return None
+               return
```

I know this is just a block ident change but since this PR makes a change on the block...